code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -228,6 +223,14 @@
 		add_movespeed_modifier(/datum/movespeed_modifier/tarantula_web)
 		silk_walking = FALSE
 
+/mob/living/simple_animal/hostile/poison/giant_spider/tarantula/RangedAttack(atom/target, modifiers)
+	. = ..()
+	if(LAZYACCESS(modifiers, RIGHT_CLICK))
+		if(COOLDOWN_FINISHED(src, charge_cooldown))
+			INVOKE_ASYNC(src, /mob/living/simple_animal/hostile/.proc/enter_charge, target)
+		else
+			to_chat(src, "<span class='notice'>Your charge is still on cooldown!</B></span>")
```

Comment:
hanging ``</B>`` tag -- determine if you meant for this to be bolded or not.

Reply:
What Bee said.